docs/hoon/syntax.md

```diff
@@ -97,100 +94,114 @@ fas /           pat @               wut ?
 zap !
 ```
 
-You don't need to memorize these glyph names, but it helps.  Code
-is often vocalized or subvocalized.   "pal" is easier to say,
-aloud or silently, then "left paren."
+You don't need to memorize these glyph names, but it might help. 
+Code is often vocalized or subvocalized. "pal" is easier to say,
+aloud or silently, than "left paren".
 
 Note that the list includes two separate whitespace forms: `ace`
-for a single space, `gap` for 2+, comment or newline.  In Hoon,
+for a single space, `gap` for 2+, comment, and newline.  In Hoon,
```

Comment:
This should be "or", `gap` is vaguely like "any of the following". But i think this might be too much parsing detail. A single space vs newlines/2-or-more is the important distinction.

Reply:
(1) New lines seem important to me, and we use them all over this doc.

(2) The sentence reads more naturally to me with the "and" but I don't have strong feelings about it.  If most disagree I'll switch it back with no fuss.

Reply:
Put another way, ace vs gap is both important and very simple. I think bringing in comments is confusing and unnecessary (I'm sure most people don't consider comments to be whitespace, and I doubt many are thinking in terms of the parsing strategy this is implicitly explaining).

But this is stylistic editorializing, maybe I should just open a PR after you land this ...

Reply:
I can live with dropping comment here.

Reply:
changed